### PR TITLE
clean imports

### PR DESCRIPTION
**Pull request by asanin-epfl**
- drop relative imports
- delete unused PointType class
- don't use private modules, rename `_neuron` to `neuron`, `_soma` to `soma`

### CODE REVIEW

**Review by arnaudon — APPROVED** [review]

**Comment by asanin-epfl** [comment]
@tomdele Alexis's approval does not allow me to merge. You will have to look here.

**Review by tomdele — APPROVED** [review]
look good

**Comment by asanin-epfl** [comment]
Thank you!

**Comment by adrien-berchet** [comment]
Hi @asanin-epfl 
No big deal but this PR introduces breaking changes due to file renamings. It would have been nice to increase the minor version number instead of just the bugfix number ;-)

**Comment by asanin-epfl** [comment]
Yes! I totally forgot. Should not be critical as private classes become public.

**Comment by adrien-berchet** [comment]
No problem, it is not critical indeed.

**Comment by asanin-epfl** [comment]
Thank for the heads up!

**Comment by arnaudon** [comment]
Wait, this could break a few other codes, no? No other complaints so far?

**Comment by asanin-epfl** [comment]
It will break only if user imported the private module.

**Comment by arnaudon** [comment]
ah ok ok , let see, it may just be me in diameter-synthesis then

**Comment by adrien-berchet** [comment]
Not only since imports were removed from the `neurom.core.__init__`.
So all `from neurom.core import *` must be changed to `from neurom.core.neuron import *`. Are these considered as private?

**Comment by asanin-epfl** [comment]
ok ok, 2.1.0

**Comment by adrien-berchet** [comment]
Thanks :-)

**Comment by adrien-berchet** [comment]
Now I am wondering if this change worth it. It actually breaks at least TNS, morph-tool and diameter-synthesis. Maybe reverting the imports in `neurom/core/__init__.py` would be simpler than updating everything else?

**Comment by asanin-epfl** [comment]
ok

**Comment by adrien-berchet** [comment]
Hard to check everything but I guess we shoud also revert `neurom/view/__init__.py` and `neurom/io/__init__.py`

**Comment by asanin-epfl** [comment]
This one no `neurom/io/__init__.py`. It duplicates already `neurom/__init__.py`.

**Comment by adrien-berchet** [comment]
But maybe another code was importing something like `neurom.io.NeuronLoader` instead of `neurom.NeuronLoader` so it would still be breaking for it. I don't think we have this kind of imports in TNS, diameter-synthesis and morph-tool so it should not bother me but maybe it's the case somewhere else so it's as you prefer.

**Comment by asanin-epfl** [comment]
yes but in that case another code needs to change such import without changing the version of NeuroM in its `setup.py`

**Comment by adrien-berchet** [comment]
Yeah true, it's ok for me then.

File: neurom/core/dataformat.py [hunk 1]
```diff
@@ -43,15 +44,9 @@ class COLS:
     XYZR = slice(0, 4)
 
 
-class POINT_TYPE:
```
Comment (tomdele) [inline neurom/core/dataformat.py:46]: [not for this review] This is so weird having this as caps .... Same for COLS

File: neurom/core/dataformat.py [hunk 2]
```diff
@@ -43,15 +44,9 @@ class COLS:
...
-    """Point types.
+Point = namedtuple('Point', ('x', 'y', 'z', 'r'))
 
-    These follow SWC specification.
-    """
-    (UNDEFINED, SOMA, AXON, BASAL_DENDRITE, APICAL_DENDRITE,
-     FORK_POINT, END_POINT, CUSTOM) = range(8)
```
Comment (tomdele) [inline neurom/core/dataformat.py:52]: [not for this review]  Same for a enum like object factorization like this are so misleading ...

**Review by tomdele — COMMENTED** [review]
looks good

**Pull request merged**